core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message.groovy

```diff
@@ -13,14 +13,18 @@ dl {
             input(name: "clear", type: "submit", value: _("Dismiss"), class: "submit-button primary")
         }
 
-        text(_("blurb"))
+        p {
+            text(_("blurb"))
+        }
```

Comment:
I added a 'p' tag wrapper to give a bit of space around the text and table. The old one was also crunched and I am not sure if this would align with the desired style. 

Without a class added it takes on a small amount of space based on the theme.